### DISCUSSION TITLE
All client side requests fail when using Ngrok with Next.js and Docker

### DISCUSSION THREAD

**Discussion by AndyBaird** [vercel/next.js#12402] [Help]
Has anyone successfully been able to use Ngrok to tunnel to a mobile device for a project with a stack that includes docker and next.js?

Our experience has been that all client-side requests fail from mobile devices and external laptops, but server-side request work properly. This is making it impossible to locally develop and test on mobile before we deploy. 

The end goal is a local development environment that allows us to test on a real phone (emulators and Chrome dev tools are not sufficient). I have had success doing this in other projects but this is the first time I have used Next.js and it does not appear to work. Any advice would be welcome.

**Comment by mehmetaydogduu** [comment]
Yes it is not working.

**Comment by lul-g** [comment]
Same here, not working. Server side works fine.

**Comment by natikos** [comment]
Any updates on this? 
it's pretty annoying that you can't test it on real phone because nextjs doesn't want to.

**Comment by just-abdullah-dev** [comment]
I have same problem did you guys found out any solution

**Comment by fabricio21x** [comment]
You should add it as a platform in your project overview page, there you can add the ngrok url as the hostname. That worked for me
